.github/workflows/gh-registry.yml

```
@@ -70,3 +70,19 @@ jobs:
         working-directory: 'hello-strongbox-npm'
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  publish-hello-strongbox-maven:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+            java-version: 1.8
+      - name: Maven install
+        run: mvn clean install
+        working-directory: 'hello-strongbox-maven'
+      # This is seperate so it may only be deployed conditionally
+      - name: Maven deploy
```

There's a missing if condition in this step:
```suggestion
      # This is seperate so it may only be deployed conditionally
      - name: Maven deploy
         if: github.ref == 'refs/heads/master'
```

---

Acutally leave it as it is right now. It might be useful for branch deploys / testing.